tests/unit/test_surrogate_th.cpp

```diff
@@ -15,10 +15,9 @@ TEST_CASE("Verify construction of surrogate thermal-hydraulics driver", "[constr
   CHECK(result);
 
   auto root = doc.document_element();
-  auto node = root.child("heat_surrogate");
+  auto node = root.child("heat_fluids");
 
-  double outlet_pressure = 101325.0;
```

Comment:
Here's the value of pressure BC that I've now set in `tests/unit/inputs/test_surrogate_input.xml`